articles/overview/overview.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Introduction to quantum computing
+description: Learn what quantum computing is, what quantum computers can do, and how you can learn quantum computing.
+author: bradben
+ms.author:  bradben
+ms.date: 05/05/2020
+ms.topic: overview
+uid: microsoft.quantum.overview.introduction
+---
+
+# Introduction to quantum computing and the Quantum Development Kit
+
+The Microsoft Quantum Development Kit (QDK) is a set of open-source tools designed to help developers learn quantum algorithms and write quantum programs. Quantum computing holds the promise to solve some of our planet's biggest challenges - in the areas of environment, agriculture, health, energy, climate, materials science, and others we haven't encountered yet.  
+
+For some of these problems, even our most powerful computers run into problems. While quantum technology is just beginning to impact the computing world, it could be far-reaching and change the way we think about computing.
+
+## What is quantum computing?
+
+In modern usage, the word quantum means the smallest possible discrete unit of any physical property, usually referring to properties of atomic or sub-atomic particles. Quantum computers use actual quantum particles, artificial atoms, or collective properties of quantum particles as processing units, and are large, complex, and expensive devices.
+
+Harnessing the unique behavior of quantum physics and applying it to computing, quantum computers introduce new concepts to traditional programming methods, making use of quantum physics behaviors such as superposition, entanglement, and quantum interference.
+
+## What can a quantum computer do?
+
+A quantum computer isn't a supercomputer that can do everything faster, but there are a few areas where quantum computers do exceptionally well.
+
+- [Quantum simulation](xref:microsoft.quantum.overview.introduction#quantum-simulation)
+- [Cryptography](xref:microsoft.quantum.overview.introduction#cryptography-and-shors-algorithm)
+- [Search](xref:microsoft.quantum.overview.introduction#search-and-grovers-algorithm)
+- [Optimization](xref:microsoft.quantum.overview.introduction#quantum-inspired-computing-and-optimization)
```

Comment:
It sounds from the xref like this is more about QIO than what a quantum computer can do, such that this line item may be better under a different header.

Reply:
This is just an in-topic link to the Optimization subsection.